ingest/Snakefile

```diff
@@ -0,0 +1,18 @@
+# USAGE: snakemake --snakefile Snakefile -c 2 pull_data
+
+from doctest import debug_script
+from snakemake.utils import min_version
+
+min_version("6.0")
+
+module vipr_workflow:
+    snakefile:
+        github("j23414/test_snakemake_6", path="workflow/vipr.smk", branch="main")
+
+
+use rule vipr_fasta from vipr_workflow as pull_data with:
+    output:
+        output="vipr_download.fasta",
+    params:
+        family='pneumoviridae',
+        virus='Respiratory%20syncytial%20virus',
```

Comment:
Based on [ZIKA.md](https://github.com/nextstrain/fauna/blob/master/builds/ZIKA.md), these are the URL parameters used there:

- datatype=genome
- family=flavi
- species=Zika virus
- fromyear=2013
- minlength=5000
- metadata=genbank,strainname,segment,date,host,country,genotype,species
- output=fasta

Can the rule be generalized to take arbitrary URL parameters, maybe as a single Snakemake param? I notice there's a slight difference between the above and [vipr.smk](https://github.com/j23414/test_snakemake_6/blob/a0e3b12eacea0cb68baeb5c4ab5ab5b8a2b7351c/workflow/vipr.smk).